tests/console/generate_dud.pm

```diff
@@ -23,14 +24,16 @@ sub run {
     my $xml = 'add_on_products.xml';
     my $dud = get_required_var('DUD');
 
-    my $content = expand_variables(get_test_data($xml));
+    my $repos = get_var('MAINT_TEST_REPO', '');
+    my $content = ($repos eq '') ? expand_variables(get_test_data($xml)) : generate_xml($repos);
     save_tmp_file($xml, $content);
     add_suseconnect_product(get_addon_fullname('phub'));
     zypper_call('in mkdud');
     assert_script_run('wget -P inst-sys ' . autoinst_url . "/files/$xml");
     assert_script_run("mkdud --create $dud --dist sle15 " .
           "--install instsys,repo --obs-keys --name 'Update' inst-sys");
     upload_asset($dud);
+    upload_logs("inst-sys/$xml");
```

Comment:
Hey Tina, could you point to me a verification run where this is uploaded?

Reply:
https://openqa.suse.de/tests/9475428  Please check: https://openqa.suse.de/tests/9475428/file/generate_dud-add_on_products.xml